Sources/Localization/Resources/en.lproj/Localizable.strings

```diff
@@ -121,7 +121,9 @@
 "ayah.menu.play-end-juz" = "to the end of Juz'";
 "ayah.menu.play-end-surah" = "to the end of Surah";
 "ayah.menu.play-end-page" = "to the end of Page";
-"ayah.menu.play-selected-verses" = "selected verses";
+"ayah.menu.play-selected-verses-once" = "selected verses (once)";
```

Comment:
I don't think "Once" adds too much value. It also increases the width of the Ayah menu, not a big deal, but something to note.

Reply:
Removed it.